### ISSUE TITLE
Disable e2e tests that are not expected to pass

### ISSUE THREAD

**Issue by surchs** [neurobagel/annotation_tool#338]
With https://github.com/neurobagel/annotation_tool/pull/330 merged, we now get the app to run correctly up to the categorization page again. That means that our integration/e2e tests for all pages up to categorization can and are passing again. However, this information is currently hidden in the summary test results because the subsequent page integration tests (annotation and download) are still failing - as expected. 

We should disable the integration tests for annotation and download pages as long as we expect them to fail, so that we can get useful summary test results during PR for the integration tests that are now expected to pass. 

In other words: e2e test should now be green again and stay like that.

**Comment by surchs** [comment]
@jarmoza: do you want to do this one?

**Comment by jarmoza** [comment]
@surchs Yes, I'll take this one one. It will be part of the phase 2 refactor, and I will have a complementary issue at the end for re-enabling the e2e tests.

**Comment by jarmoza** [comment]
Closed by #343
